deploy/roles/beaker/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+beaker_mirror:              "http://d299yghl10frh5.cloudfront.net/beaker-notebook-1.7.1-0-g6dac09a-ubuntu.zip"
+beaker_src_dir:             "/usr/local/src"
+beaker_parent_dir:          "/usr/lib"
```

Comment:
Let's use /opt instead.  Normally, we've been using /usr/local for applications installed via Yum, and /opt for applications that are extracted directly from an archive.